packages/integrations/cloudflare/README.md

```diff
@@ -48,7 +48,9 @@ Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode
 
 For most projects the adapter default of `advanced` will be sufficient; the `dist` folder will contain your compiled project. Switching to directory mode allows you to use [pages plugins](https://developers.cloudflare.com/pages/platform/functions/plugins/) such as [Sentry](https://developers.cloudflare.com/pages/platform/functions/plugins/sentry/) or write custom code to enable logging.
 
-In directory mode the adapter will compile the client side part of your app the same way, but moves the worker script into a `functions` folder in the project root. The adapter will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control. Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
+In directory mode the adapter will compile the client side part of your app depending on the new `build.split` setting. If `split` is disabled the adapter compiles everything in the same way, but moves the worker script into a `functions` folder in the project root. The adapter will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control.
+
+`Experimental:` if `split` is enabled the adapter compiles for each pages a seperate bundle, which is used by cloudflare routing. Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
```

Comment:
```suggestion
With the build configuration `split: true`, the adapter instead compiles a seperate bundle for each page which is used by cloudflare routing. Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
```
I think this might be clear enough without needing to overload the first paragraph with information about `build.split`, since people might not even set that configuration?

If this is determined to be experimental (not just "new, and I hope it doesn't break" but rather behind a flag), then ping Docs again to determine the appropriate syntax for marking as experimental!